recipe/bld.bat

```diff
@@ -1,22 +1,23 @@
+set PKG_CONFIG_PATH=%LIBRARY_PREFIX%\lib\pkgconfig; 
 
 mkdir build
 cd build
 
 cmake ../^
     -GNinja^
     -DCMAKE_BUILD_TYPE=Release^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%"^
     -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"^
-    -DINCLUDE_PREFIX="%LIBRARY_INC%"^
-    -DCMAKE_PREFIX="%LIBRARY_PREFIX%"^
-    -DLIB_PREFIX="%LIBRARY_LIB%"^
-    -DBIN_PREFIX="%LIBRARY_BIN%"^
-    -DWITH_SELFCONTAINED=OFF^
+    -DINCLUDE_PREFIX=include^
+    -DCMAKE_PREFIX=lib/cmake/casadi^
+    -DLIB_PREFIX=lib^
+    -DBIN_PREFIX=bin^
     -DWITH_SELFCONTAINED=OFF^
     -DWITH_PYTHON=ON^
     -DWITH_PYTHON3=ON^
     -DWITH_LAPACK=ON^
     -DWITH_IPOPT=ON^
-    -DWITH_THREAD=ON^
+    -DWITH_THREAD=OFF^
```

Comment:
The python prefix should point to site package, and not root, you can have a look at the build.sh

Reply:
Fixed in https://github.com/conda-forge/casadi-feedstock/pull/47/commits/48575e87853e3f09ad5315637d8a5e470a3f40ac .